package.json

```diff
@@ -12,20 +12,23 @@
         "build": "tsc",
         "format": "prettier --write \"**/*.{ts,tsx,json,md}\"",
         "prepublishOnly": "rm -rf ./dist && npm test && npm run lint && npm run build",
-        "build-storybook": "storybook build"
+        "build-storybook": "storybook build",
+        "chromatic": "npx chromatic --exit-zero-on-changes"
```

Comment:
what does `--exit-zero-on-changes` give us?

Reply:
It would allow us to us the UI Tests and UI Review features as described here in the [Configure CI docs](https://www.chromatic.com/docs/ci#configure-ci)